nservicebus/azure-service-bus/performance-tuning_body_asb_[7,).partial.md

```diff
@@ -0,0 +1,61 @@
+## Tuning send performance outside the scope of a handler
+
+Only settings at the ASB SDK level impact this scenario, more specifically
+
+  * `MessagingFactories().BatchFlushInterval()`: Controls the time that the ASB SDK batches client side requests before sending them out. A value of TimeSpan.Zero turns this feature off. Values between 100 and 500 milliseconds appear to work best.
```

Comment:
> A value of TimeSpan.Zero turns this feature off. Values between 100 and 500 milliseconds appear to work best.

Optimal values are between 100 and 500 milliseconds. To turn this feature off, provide `TimeSpan.Zero`.